site/src/docs/components/highlight/index.mdx

```diff
@@ -0,0 +1,78 @@
+---
+slug: '/components/highlight'
+title: 'Highlight'
+navTitle: 'Highlight'
+---
+
+import { Highlight } from 'hds-react';
+import ExternalLink from '../../../components/ExternalLink';
+import PlaygroundPreview from '../../../components/Playground';
+import TabsLayout from './tabs.mdx';
+import InternalLink from '../../../components/InternalLink';
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## Usage
+
+### Example
+
+<PlaygroundPreview>
+  <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent ultrices tincidunt nunc, porttitor pretium enim vestibulum eget. Suspendisse ac tempor ante. Nullam at convallis ex. Integer mattis sagittis interdum. Suspendisse viverra sit amet mauris non interdum. In porttitor tempus diam vel venenatis. Aenean elementum sem vitae tempus tempor. </p>
+  <Highlight text="You may select an highlight from the article to be displayed here. Select an excerpt that you want the user to pay attention to." />
+  <p>Sed quis massa elementum, pulvinar lacus at, efficitur velit. Nam nec ornare augue. Aliquam pellentesque dictum nisl, a commodo arcu dapibus id.</p>
+  <Highlight type="quote" text="Add an interesting quote here" reference="First name Last name. Title."/>
+  <p>Sed mollis mauris nec dui condimentum pulvinar. Suspendisse vitae aliquet nisi. Donec libero ipsum, fermentum ut augue id, scelerisque lobortis orci.</p>
+</PlaygroundPreview>
+
+### Principles
+
+- Highlights are designed to capture users' attention. ** Therefore, they should be used sparingly. **
+- Use the prop `type="quote" ` to transform the highlight text into a quote with an optional reference, though we strongly recomend adding a reference to a quote
+- Pay attention to colours when using custom text or accent line color
```

Comment:
Pay attention how? 🤔 In regards to accessibility? If so then Accessibility tab might be a better place for this with a little more information here. Like "Pay attention that colours are accessible when --"